rasa_core/policies/ensemble.py

```diff
@@ -315,17 +328,30 @@ def probabilities_using_best_policy(self,
         result = None
         max_confidence = -1
         best_policy_name = None
+        best_policy_priority = -1
 
         for i, p in enumerate(self.policies):
             probabilities = p.predict_action_probabilities(tracker, domain)
+
             if isinstance(tracker.events[-1], ActionExecutionRejected):
                 probabilities[domain.index_for_action(
                     tracker.events[-1].action_name)] = 0.0
             confidence = np.max(probabilities)
+
             if confidence > max_confidence:
                 max_confidence = confidence
                 result = probabilities
                 best_policy_name = 'policy_{}_{}'.format(i, type(p).__name__)
+                best_policy_priority = p.priority
+
+            # higher priority wins when confidence equal
+            elif confidence == max_confidence:
```

Comment:
this code should not be duplicated, should rather be an or in the `if` in L341

Reply:
so then should it look like this?
```
if (confidence > max_confidence or (confidence == max_confidence and p.priority > best_policy_priority)
```

Reply:
i think this should work as well
```python
if (confidence, p.priority) > (max_confidence, best_policy_priority):
```

Reply:
Yep, tested it and it does have the functionality we want, i can implement that. How does that syntax work exactly? Does it compare the first and if they're equal move on to comparing the second? I haven't seen this and am having trouble looking it up

Reply:
it is pythons normal way to compare tuples